http_client.rst

```diff
@@ -481,8 +482,10 @@ each request (which overrides any global authentication):
 .. note::
 
     The NTLM authentication mechanism requires using the cURL transport.
-    By using ``HttpClient::createForBaseUri()``, we ensure that the auth credentials
-    won't be sent to any other hosts than https://example.com/.
+    By using the ``createForBaseUri()`` method of
+    :class:`Symfony\\Component\\HttpClient\\HttpClient`, we ensure that
```

Comment:
I think the current one is more readable

Reply:
Alright, reverted :slightly_smiling_face: